### PR TITLE
NSLocalizedString stub

### PR DESCRIPTION
**Pull request by johnno1962**
For separate review

### CODE REVIEW

**Comment by alblue** [comment]
Looks good, same implementation as in https://github.com/apple/swift/blob/485ebefba7ae044c85bcbd4ebf48bfd18c807c2f/stdlib/public/SDK/Foundation/Foundation.swift#L34-L40

@swift-ci please test and merge

File: Foundation/NSString.swift [hunk 1]
```diff
@@ -20,6 +20,16 @@ extension unichar : ExpressibleByUnicodeScalarLiteral {
     }
 }
 
+// Placeholder for a future implementation
```
Comment (ianpartridge) [inline Foundation/NSString.swift]: Is this comment correct? Haven't you provided the implementation?

Comment (johnno1962) [reply Foundation/NSString.swift]: I’ll remove it from the rebase.

Comment (alblue) [reply Foundation/NSString.swift]: The comment from the SDK implementation reads:

/// Returns a localized string, using the main bundle if one is not specified.

Maybe we can use that instead?

Comment (johnno1962) [reply Foundation/NSString.swift]: Sure, added to the rebase

**Pull request merged**